### ISSUE TITLE
why did you set difficult as False?

### ISSUE THREAD

**Issue by shwoo93** [amdegroot/ssd.pytorch#53]
Hello 

I am wondering why did you set the difficult training set as False.
Since I found that original code uses difficult training set as well.

Thanks

**Comment by amdegroot** [comment]
Hi, this is a good point.  I will see if the results differ with difficult training set to True.  We were able to reproduce the mAP without difficult training though oddly.

**Comment by shwoo93** [comment]
It seems the code trains the network(SSD) without difficult training sets and evaluates with out difficult test sets. This seems unfair to compare the final mAP of this code with the original caffe code which uses difficult sets for both trianing and testing.

Additionally I also trained the network with 07 ++ 12 train set (07 trainval, 07 test, 12 trainval) and tested with 12 test set using the official server. And the result was 74.1% 2% below from the latest version of SSD300(75.8%).  Since the server's ground truth would contain difficult sets, but the network which was trained only with easy sets would not be able to achieve the original performance.
